### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by stevebanik**
Fixes #

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: DDKIntegrationTestsD8472609-5SCHWjldmWcf
* Commit ID: c055035379d0cf2576e46801e56c683b4b7e3cfc
* Result: SUCCEEDED
* [Build Logs](https://9hm1y4sor9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=af8853cc-ef39-42c7-bbae-88d26e1c7887%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by malachi-constant — APPROVED** [review]

File: docs/release/1.0.0/how-to/upgrade-guide.markdown [hunk 1]
```diff
@@ -16,7 +16,7 @@ Version `1.0.0` of the AWS DDK includes the following high-level changes and upd
   - [`SnsSqsToLambdaStage()`](https://constructs.dev/packages/aws-ddk-core/v/1.0.0-beta.1/api/SnsSqsToLambdaStage)
 
 ## Config
-`Configurator` is the new **optional** construct used to manage config across multiple environments. While this can handle a variety of use cases we encourage users to build their own config mechanisms when it `Configurator` is not the best option. See [this how-to guide](https://awslabs.github.io/aws-ddk/release/latest/how-to/ddk-configuration.html)for more on using Configurator.
+`Configurator` is the new **optional** construct used to manage config across multiple environments. While this can handle a variety of use cases we encourage users to build their own config mechanisms when it `Configurator` is not the best option. See [this how-to guide](https://awslabs.github.io/aws-ddk/release/latest/how-to/ddk-configuration.html) for more on using Configurator.
```
Comment (malachi-constant) [inline docs/release/1.0.0/how-to/upgrade-guide.markdown:19]: Hey @stevebanik thanks. Can you make this change [here](https://github.com/awslabs/aws-ddk/blob/main/docs/release/latest/how-to/upgrade-guide.markdown). That way it gets propogated to all new versions.

**Review by malachi-constant — CHANGES_REQUESTED** [review]

File: docs/release/1.0.0/how-to/upgrade-guide.markdown (hunk 1 above)
Comment (stevebanik) [reply docs/release/1.0.0/how-to/upgrade-guide.markdown:19]: @malachi-constant ref [this pr](https://github.com/awslabs/aws-ddk/pull/483)

**Comment by malachi-constant** [comment]
Moved to #483

**Pull request closed without merging**